components/src/lib/components/Portal.tsx

```
@@ -0,0 +1,35 @@
+  /**
+   * The content of the modal
+   */
+  children: React.ReactNode;
+} & PropsWithChildren;
+
+export const Portal = ({ isOpen, children }: Props) => {
+  if (!isOpen) return null;
+
+  return (
+    <PortalWrapper>
+      <>{isOpen && children}</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)